.github/workflows/pythonpackage.yml

```diff
@@ -26,8 +26,9 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
+        pip install --no-binary=:all: pyzmq
         pip install flake8 pytest
-        pip install .
+        pip install --editable .
```

Comment:
What is the reason to do the editable here? (just asking for a friend)

Reply:
The github tests were failing without it because the local directory was somehow shadowing the global installation and therefore the cython extension module couldn't be successfully imported.

I'm sure there's a better way to solve that problem, but for now I think we should leave editable in the tests because that's how we are recommending users install the package.

Reply:
that's fine, let's keep it that way for now then